client/my-sites/domains/domain-management/transfer/transfer-page/index.jsx

```diff
@@ -12,11 +18,39 @@ import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-t
 import { hasLoadedSiteDomains } from 'calypso/state/sites/domains/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
-const TransferPage = () => {
+import './style.scss';
+
+const TransferPage = ( { selectedSite, selectedDomainName, currentRoute, translate } ) => {
 	return (
-		<Main wideLayout>
+		<Main wideLayout className="transfer-page">
 			<BodySectionCssClass bodyClass={ [ 'edit__body-white' ] } />
-			The page goes heres
+			<Card>
+				<ActionCard
+					buttonHref={ domainManagementTransferToAnotherUser(
+						selectedSite.slug,
+						selectedDomainName,
+						currentRoute
+					) }
+					buttonText={ translate( 'Continue', { context: 'Verb' } ) }
+					headerText={ translate( 'To another user', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Transfer to another user' )` *ES Score: 6*